jetson/teleop/src/__main__.py

```diff
@@ -114,6 +114,41 @@ def gimbal_control_callback(channel, msg):
     lcm_.publish('/mast_gimbal_cmd', gimbal_msg.encode())
 
 
+class Camera:
+    def __init__(self):
+        self.cameras = [-1, -1]
+        self.ish_cameras = [-1, -1]
+
+    def update_cameras(self, channel, msg):
+        message = GUICameras.decode(msg)
+
+        if message.port[0] == -1 or message.port[0] not in self.ish_cameras:

```

Comment:
you'll be able to get rid of this logic now right? bc it can send the same stream to multiple ports?